linux/step01_centos7_ice_venv.sh

```diff
@@ -5,13 +5,13 @@ set -e -u -x
 VENV_SERVER=${VENV_SERVER:-/opt/omero/server/venv3}
 #start-ice-py
 # Create a virtual env
-python3 -mvenv $VENV_SERVER
+python -mvenv $VENV_SERVER
 
 # Upgrade pip
 $VENV_SERVER/bin/pip install --upgrade pip
 
 # Install the Ice Python binding
-$VENV_SERVER/bin/pip install https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.2.1/zeroc_ice-3.6.5-cp36-cp36m-linux_x86_64.whl
+#$VENV_SERVER/bin/pip install https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.2.1/zeroc_ice-3.6.5-cp36-cp36m-linux_x86_64.whl
```

Comment:
Is the issues  that we have not built a wheel yet for testing or is a wheel not an option in this scenario?

Reply:
we have to build a wheel which is something we may want to reconsider.
This obviously has an impact on the dependencies required e.g. openssl-devel, python-devel etc.

Reply:
c2cacd9 uses the wheel